daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala

```diff
@@ -64,25 +64,25 @@ object Main extends App {
     System.exit(1)
   } else {
     var replArgs = args.toList
-    var allowDev = false
+    var devMode = false
     replArgs match {
-      case "--decode-lfdev" :: rest =>
+      case "--dev" :: rest =>
```

Comment:
Maybe time to switch to a proper CLI parser?

Reply:
maybe in another PR.